packages/components/src/RichTextEditor/RichTextEditor/_docs/RichTextEditor.stories.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from "react"
+import { Meta, StoryObj } from "@storybook/react"
+import { EditorContentArray } from "~components/RichTextEditor/types"
+import { RichTextEditor, RichTextEditorProps } from "../index"
+
+const meta = {
+  title: "Components/RichTextEditor",
+  component: RichTextEditor,
+  parameters: {
+    chromatic: { disable: false },
+  },
+  args: {
+    labelText: "Rich Text",
+    defaultValue: [],
+    onChange: () => undefined,
+  },
+} satisfies Meta<typeof RichTextEditor>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const Playground: Story = {
+  render: args => {
+    const [rteData, setRTEData] = useState<EditorContentArray>([])
+    const handleOnChange: RichTextEditorProps["onChange"] = (
+      editorState
+    ): void => setRTEData(editorState.toJSON().doc.content)
+
+    return (
+      <RichTextEditor
+        {...args}
+        defaultValue={rteData}
+        onChange={handleOnChange}
+      />
+    )
+  },
+  parameters: {
+    docs: {
+      canvas: {
+        sourceState: "shown",
```

Comment:
One call out that Dale mentioned a while back was that when he first used the RTE, it was that it wasn't clear to him that `handleOnChange` would need to setter with `editorState.toJSON().doc.content`.

Something we could either document later or expose in the source code of the canvas

Reply:
I can do a hardcode here again 👍